Journaley/Utilities/PostMarkdownParser.cs

```diff
@@ -0,0 +1,112 @@
+﻿namespace Journaley.Utilities
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Text.RegularExpressions;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Helper class for fixing MarkdownDeep parsed HTML.
+    /// </summary>
+    public class PostMarkdownParser
+    {
+        /// <summary>
+        /// Perform fixes after MarkdownDeep parsing for publishing
+        /// to Journaley.
+        /// - automatically adds break tags on single line breaks.
+        /// - adds strikethrough style support.
+        /// - makes the first sentence into a header.
+        /// </summary>
+        /// <param name="formattedString">Formatted HTML string</param>
+        /// <returns>Properly formatted HTML string for publishing.</returns>
+        public static string PostMarkdown(string formattedString)
+        {
+            StringBuilder builder = new StringBuilder();
+            StringBuilder paragraphBuilder = new StringBuilder();
+
+            // 0 - stumbles upon the beginning of a <p> tag/usual parsing.
+            // 1 - stumbles upon the end of </p> tag.
+            // 2 - skips check and just dumps the line to builder.
+            var parseState = -1;
+
+            bool firstTitleParsed = false;
```

Comment:
I'm sorry, but could you make the firstTitle feature as a separate PR, once this one is merged? This seems to require more thoughts on it.
Also, it seems like the current implementation would convert the first single line sentence into a title, even if it's not at the top.